### PR TITLE
Update installing-index.rst

### PR DESCRIPTION
**Pull request by stj-fr**
Hello 
I have done a typo fix, the version number in the path was different.

### CODE REVIEW

File: source/install-guide/installing-index.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ Stop Elasticsearch using
 
 .. code-block:: shell
 
-    elasticsearch-7.6.2/bin/elasticsearch stop
+    elasticsearch-7.9.2/bin/elasticsearch stop
```
Comment (jodygarnett) [inline source/install-guide/installing-index.rst:34]: does this represent the latest tested version? or should we rewrite this to indicate to use the latest.

Comment (jodygarnett) [reply source/install-guide/installing-index.rst:34]: never mind. i see you wanted to be consistent with the above instructions.

**Review by jodygarnett — APPROVED** [review]

**Comment by jodygarnett** [comment]
This PR was still open? It missed out on the mkdocs conversion ... can you make this change to core-geonetwork please

**Comment by stj-fr** [comment]
Hello Jody,
i will close it and edit core-geonetworks
Kind regards

**Pull request closed without merging**